wp-content/themes/core/components/comments/comment/Comment_Controller.php

```diff
@@ -14,17 +14,19 @@ class Comment_Controller extends Abstract_Controller {
 
 	use Comment_Edit_Link;
 
-	public const COMMENT_ID = 'comment_id';
-	public const CLASSES    = 'classes';
 	public const ATTRS      = 'attrs';
+	public const CLASSES    = 'classes';
+	public const COMMENT_ID = 'comment_id';
 	public const DEPTH      = 'depth';
 	public const MAX_DEPTH  = 'max_depth';
+	public const STYLE      = 'style';
```

Comment:
In other components, we've opted to use the `TAG` const and `$tag` property to denote the HTML tag to use for a component. (Reference the Text block as a example.) I'd prefer to keep that syntax in the component as well for consistency.

Reply:
This is actually the parent tag that is defined as the style attribute when calling the walker. I didn't want to add confusion since it isn't a 1:1 with the child tag.  The style attribute is either ul, ol, or div.

Reply:
Still, it's not a "style" it's an it's an html tag. I'd prefer to keep the component consistent with other components. If we're rendering a html tag, then let's use `TAG` and 

```
public function get_tag(): string {
	return tag_escape( $this->tag );
}
```

Instead of trying to deduce the tag based on a parent tag/markup.